src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -67,6 +67,18 @@ public interface IAdapterSettings
         void RestoreRandomSeed(string dirname);
     }
 
+
+    public interface IFeatureFlags
+    {
+        bool UseTestCaseFilterConverter { get; }
+    }
+
+    public class FeatureFlags : IFeatureFlags
```

Comment:
Is this alternate implementation of IFeatureFlags public by design, or can it be private and maybe even a struct?
I have to think about why both AdapterSettings and FeatureFlags implement IFeatureFlags and where in the codebase FeatureFlags might be used versus AdapterSettings.

Reply:
This FeatureFlags class is used for the runsettings, so that we can separate out this block of settings.  Since the other ones are options, so to speak, I find the featureflags different, and therefore easier to have them in a separate block.  In the code though, it is enough that we can see them in the adaptersettings, therefore the interface is implemented there too, and implementation just passed down.